src/schema/nmdc.yaml

```diff
@@ -140,6 +140,16 @@ subsets:
       - Assignment of the data_portal_subset is incomplete in the schema.
 
 classes:
+  EukEval:
+    description: This class contains information pertaining to evaluating if a Metagenome-Assembled Genome (MAG) is eukyarotic.
+    comments:
+     - A tool like eukCC (https://doi.org/10.1186/s13059-020-02155-4) would generate information for this class.
+    slots:
+     - completeness
+     - contamination
+     - ncbi_lineage_taxIDs
```

Comment:
actually, I did find something that requires changes: slots names should always be pure lower_snake_case